lms/static/sass/multicourse/_courses.scss

```diff
@@ -93,7 +93,6 @@ $facet-background-color: #007db8;
       height: 120px;
       margin: 0 auto;
       max-width: 1200px;
-      min-width: 760px;
```

Comment:
@frrrances is working on a separate PR that removes these as well. (https://github.com/edx/edx-platform/pull/8309) No action necessary from you here Frances, but I just wanted to share with @dsego a link to your PR. 

@dsego would you mind reviewing her PR to make sure the approach is consistent with her work here? For example, it looks like some of the rules changed in this PR might alter the dashboard, global footer, etc. We'll want to make sure we test all the pages impacted by this change. 

Thanks!